### DISCUSSION TITLE
Starter template for vuejs

### DISCUSSION THREAD

**Discussion by farazshuja** [crxjs/chrome-extension-tools#516] [General]
Hi,

I have just created a starter template for vuejs using crxjs and like to share so others can also contribute and extend it. Along with crxjs I also added following
- Shadow-dom for content script components. so can use css libs without colliding with page
- Webext Bridge - Messaging in WebExtension made super easy. Out of the box.
- Windi CSS - on-demand CSS utilities
- Components auto importing
- Icons - Access to icons from any icon set directly

https://github.com/farazshuja/chrome-extension-vite-vue-starter-template

thx

**Comment by jacksteamdev** [comment]
@farazshuja Looks amazing! 🚀 

I made a pinned discussion for [Awesome CRXJS](https://github.com/crxjs/chrome-extension-tools/discussions/81) projects. You are welcome to post a link there as well.
